collectd.conf

```diff
@@ -124,7 +124,10 @@ PostCacheChain "PostCache"
   </Rule>
 </Chain>
 
-# if you have other config, especially for plugins, you can drop them
-# into this directory
+# Obsolete config directory for backward compatibility.
```

Comment:
Users could have put files into the other dir without the `.conf` extension, right? Since that's already not totally backwards-compatible (changing that one to take `.conf` only), I'd say for consistency we should either:
1. get rid of 2 dir options and just leave the new block including `.conf` (131-133), or
2. have both dir options, and they both only accept `.conf` files

(i'm partial to the former)